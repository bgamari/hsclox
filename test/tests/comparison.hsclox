print "hi"=="hi";
print "hi"=="bye";
print 909>=606;
print 909>606;
print 909==606;
print 909<=606;
print 909<606;
print true!=true;
print false!=true;
print true!=false;
print false!=false;
print true!=true;
print false!=true;
print true!=false;
print false!=false;
