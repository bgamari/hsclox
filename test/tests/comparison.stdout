== code ==
0000    1 OP_CONSTANT         0 '909
0002    | OP_CONSTANT         1 '606
0004    | OP_LESS
0005    | OP_NOT
0006    2 OP_RETURN
true
